cmd/apps.go

```diff
@@ -115,6 +119,14 @@ var cmdAppsCreateOneLogin = &cobra.Command{
 			"app-id":   appID,
 			"provider": provider,
 		}
+
+		if duration != 0 {
```

Comment:
Why do we need to check if duration is not zero? `if duration < 3600 || duration > 43200` already verifies that, unless I am missing something.

Reply:
the default if unspecified is 0. Plan was to not the pollute the config file with default values.

Reply:
I understand. Doesn't Viper have a handling for not writing default values to YAML files? In any case, if we do this kind of handling we should open issues for similar things in other places, for example this one: 

https://github.com/allcloud-jonathan/clisso/blob/baae94064f254a0037a12ae459609e6a288549e7/cmd/providers.go#L113

Reply:
No, it doesn't. Yes, that should be added I guess.